src/components/CreateUser/CreateUser.scss

```diff
@@ -0,0 +1,49 @@
+.create-user-form label {
+    display: inline-block;
+    margin: 0;
+}
+
+.create-user-form input {
+    vertical-align: top;
+    margin: 0;
+}
+
+.user-input label {
+    display: inline-block;
+    width: 150px;
+    font-weight: bold;
+    margin-right: 10px;
+    text-align: right;
+}
+
+.input-field {
+    display: inline-block;
+    width: calc(100% - 160px);
+    padding: 10px;
+    font-size: 16px;
+    border-radius: 5px;
+    border: none;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+}
+
+.create-user-submit {
+    color: white;
```

Comment:
Do we want to use a colour from the constants file here?

Reply:
changed.